spec/use_cases/administrator/sort_users_spec.rb

```diff
@@ -0,0 +1,26 @@
+describe UseCases::Administrator::SortUsers do
```

Comment:
can there also be a test to ensure values fetched from the database get sorted?
This allows us to assert ordering outside of a feature test

Reply:
I guess what you are talking about is the integration test that ties this all together. It currently exists in a few feature specs.

Reply:
Done @cybojenix